sphinx/source/index.rst

```diff
@@ -1,6 +1,8 @@
 Welcome to Bokeh
 ================
 
+**A framework for interactive browser-based visualizations**
+
```

Comment:
Sorry I meant project! It's a standard project descriptor, not product. Please let me know what you and Michele resolve about this.